native_client/Makefile

```diff
@@ -1,14 +1,41 @@
+###
+### From topdir, first use multistrap to prepare a raspbian jessie armhf root
+### $ sudo multistrap -d multistrap-raspbian-jessie -f native_client/multistrap.conf
+###
+### You can make a tarball after:
+### $ touch multistrap-raspbian-jessie.tar && sudo tar cf multistrap-raspbian-jessie.tar multistrap-raspbian-jessie/ && xz multistrap-raspbian-jessie.tar
+###
+### Then cross-build:
+### $ make -C native_client/ TARGET=rpi3 TFDIR=../../tensorflow/tensorflow/
+###
 
-TFDIR ?= ../../tensorflow
-CFLAGS ?= -O2 -Wall
+TARGET ?= host
+TFDIR  ?= ../../tensorflow
+CXX    ?= c++
+
+ifeq ($(TARGET),host)
+TOOLCHAIN    :=
+HOST_CFLAGS  := `pkg-config --cflags sox`
+HOST_LDFLAGS := `pkg-config --libs sox`
```

Comment:
Having HOST_CFLAGS and TARGET_CFLAGS both representing the target cflags is a bit confusing. If target==host, then why not just set TARGET_CFLAGS/LDFLAGS here and use those later on instead of adding HOST and TARGET flags into CFLAGS? I think this is unnecessarily mixing up HOST/TARGET.